src/backends/github/API.js

```diff
@@ -386,7 +386,7 @@ export default class API {
     return this.deleteRef("heads", branchName);
   }
 
-  createPR(title, head, base = "master") {
+  createPR(title, head, base = this.branch) {
```

Comment:
Just tested -- I still get the error if I change it back to `base = "master"` and use `this.createPR(options.commitMessage, branchName, this.branch)`, so it seems not